hexkit/providers/akafka/testutils.py

```diff
@@ -364,6 +365,31 @@ def record_events(self, *, in_topic: Ascii) -> EventRecorder:
 
         return EventRecorder(kafka_servers=self.kafka_servers, topic=in_topic)
 
+    def delete_topics(self, topics: Optional[Union[str, list[str]]] = None):
+        """Delete given topic(s) from Kafka broker.
+
+        This process deletes the contained messages as the topics will be recreated
+        by the default config.
+        """
+
+        if topics is None:
+            topics = []
+        elif isinstance(topics, str):
+            topics = [topics]
+        admin_client = KafkaAdminClient(bootstrap_servers=self.kafka_servers)
+        try:
+            existing_topics = set(admin_client.list_topics())
```

Comment:
```suggestion
        admin_client = KafkaAdminClient(bootstrap_servers=self.kafka_servers)
        all_topics = admin_client.list_topics()
        if topics is None:
            topics = all_topics
        elif isinstance(topics, str):
            topics = [topics]
        try:
            existing_topics = set(all_topics)
```

Reply:
When None is passed, we want to start on a clean slate and remove all topics.

Reply:
Good idea. Change is part of [b7d4826](https://github.com/ghga-de/hexkit/pull/59/commits/b7d482637e1285499a7258a4b9f44ad3b2b63bcc)